data/geolocation-!cn

```diff
@@ -58,8 +58,9 @@ include:stackpath
 # Certificate authorities
 include:category-cas
 
-# Communications
-include:category-communication
+# Communication
+include:ntt
+include:softbank
```

Comment:
Not all sites of softbank are related to communication, I guess

Reply:
i reverted this change